sdk_container/src/third_party/coreos-overlay/profiles/coreos/targets/generic/dev/profile.bashrc

```diff
@@ -0,0 +1,29 @@
+# Adapted from cross-boss. Only needed for the dev profile because we don't
+# build the SDK with sysroot flags and we don't install build tools to the
+# production image.
+cros_pre_pkg_preinst_strip_sysroot() {
+	local FILES FILE
+
+	# Gather all the non-binary files with the --sysroot or --with-sysroot flag.
+	mapfile -t -d '' < <(find "${D}" -type f -exec grep -lZEIe "--sysroot=${ROOT}\b|--with-sysroot=${EROOT}\b" {} +) FILES
+
+	# Only continue if there are any.
+	[[ ${#FILES[@]} -eq 0 ]] && return
+
+	einfo "Stripping sysroot flags from:"
+	for FILE in "${FILES[@]}"; do
+		einfo "  - ${FILE#${D}}"
+	done
```

Comment:
I was thinking if passing `"${FILES[@]#${D}}"` to `einfo` would to the trick, but then you would still need to prepend the `  -`, so you would still need some intermediate variable anyway. Meh, nevermind.